compiler/src/iree/compiler/Codegen/Common/MathTransformPass.cpp

```diff
@@ -124,6 +123,9 @@ static bool predicateApprox(StringRef name,
   StringRef expm1 = math::ExpM1Op::getOperationName();
   StringRef cbrt = math::CbrtOp::getOperationName();
   StringRef erf = math::ErfOp::getOperationName();
+  if (target.getBackend() == "rocm" && name == erf) {
```

Comment:
You should be able to use this https://github.com/iree-org/iree/blob/main/compiler%2Fsrc%2Firee%2Fcompiler%2FCodegen%2FUtils%2FUtils.h#L66